src/main/java/Arsdorint/Arsdorint.java

```diff
@@ -0,0 +1,336 @@
+        try {
+            File file = new File(STORAGE_FILE_NAME);
+            Scanner fileRead = new Scanner(file);
+            showToUser(MESSAGE_LOAD_FILE);
+            while (fileRead.hasNext()) {
+                fileParser(fileRead.nextLine());
+            }
+            fileRead.close();
+        } catch (FileNotFoundException err) {
+            showToUser(MESSAGE_NO_FILE);
+        } catch (ArrayIndexOutOfBoundsException err) {
+            showToUser(MESSAGE_WRONG_FILE);
+        }
+    }
+
+    private static void fileParser(String newLine) {
+        String[] parsedLine = newLine.split("");
+        switch (parsedLine[0]) {
+            case Todo.typeToDo:
+                toDoList.add(new Todo(parsedLine[2]));
+                toDoList.get(Task.numOfTasks - 1).isDone = Boolean.valueOf(parsedLine[1]);
+                addTaskMessage();
+                break;
+            case Deadline.typeDeadline:
+                toDoList.add(new Deadline(parsedLine[2], parsedLine[3]));
+                toDoList.get(Task.numOfTasks - 1).isDone = Boolean.valueOf(parsedLine[1]);
+                addTaskMessage();
+                break;
+            case Event.typeEvent:
+                toDoList.add(new Event(parsedLine[2], parsedLine[3]));
+                toDoList.get(Task.numOfTasks - 1).isDone = Boolean.valueOf(parsedLine[1]);
+                addTaskMessage();
+                break;
+        }
+    }
+
+    private static void exitMessage() {
+        showToUser(MESSAGE_DIVIDER, EXIT_MESSAGE, MESSAGE_DIVIDER);
+    }
+
+    private static void showToUser(String... message) {
+        for (String i : message) System.out.println(i);
+    }
+
+    private static void helloMessage() {
+        System.out.println("Hello from\n" + logo);
+        showToUser(MESSAGE_DIVIDER, HELLO_MESSAGE, COMMAND_LIST_MESSAGE, QUESTION, MESSAGE_DIVIDER);
+    }
+    private static void command() {
+        String command = input.nextLine();
+        String[] splitLine = command.split(" ");
+        String lowerCaseLine = splitLine[0].toLowerCase();
+        if (lowerCaseLine.equals("bye")) exit();
+        else if (lowerCaseLine.contains("list")) {
+            System.out.println("Here are the tasks in your list:\n");
+            list();
+        }
+        else if (lowerCaseLine.equalsIgnoreCase("mark")) {
+            mark(splitLine);
+        }
+        else if (lowerCaseLine.equalsIgnoreCase("unmark")) {
+            unmark(splitLine);
+        }
+        else if (lowerCaseLine.equalsIgnoreCase("todo") ||
+                lowerCaseLine.equalsIgnoreCase("deadline") ||
+                lowerCaseLine.equalsIgnoreCase("event")) add(command);
+            //add command when user don't type the instruction's TaskList.command
+        else if (lowerCaseLine.equalsIgnoreCase("delete")) {
+            delete(splitLine);
+        }
+        else if (lowerCaseLine.equalsIgnoreCase("save")) {
+            save();
+        }
+        else wrongCommandMessage();
+    }
```

Comment:
Consider re-organising this if-else block to match the format stated in the coding standard, in order to maintain consistnecy.